stories/topics/proc-gcp-backup-requirements.adoc

```diff
@@ -0,0 +1,28 @@
+[id="proc-gcp-backup-requirements"]
+
+= Backup requirements
+
+A bucket must be created in order to store the Ansible on Clouds deployment backup.
```

Comment:
You must create a bucket to store the ...

Reply:
Done